src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -4928,6 +4966,16 @@ namespace InternalHeatGains {
                 } else {
                     print(state.files.eio, "No\n");
                 }
+                if (People(Loop).CoolingEffectASH55) {
+                    print(state.files.eio, "Yes\n");
+                } else {
+                    print(state.files.eio, "No\n");
```

Comment:
Only the last group should have `\n` (line break) so it should be removed here and in the KSU block above.

Reply:
Fixed.